### ISSUE TITLE
Polish diacritics

### ISSUE THREAD

**Issue by ms-studio** [ms-studio/karmilla#9]
As per popular request:

"Would it be possible to add the latin extended characters found in Polish? I’m thinking about ę ą ó ł ś ć ń ż ź and their capital counterparts."

**Comment by ms-studio** [comment]
We currently have all of them in the Regular cut!

Still need to add ę ą and ł to the Bold version.

**Comment by andrewshadura** [comment]
The Bold version is also missing some Slovak letters.

**Comment by ms-studio** [comment]
A task for a rainy Sunday sometime soon ? :)

**Comment by andrewshadura** [comment]
I'd rename this to "Polish the diacritics", because I found out even the existing diacritics in the bold font need some polishing :)

**Comment by andrewshadura** [comment]
Any news on this?

**Comment by andrewshadura** [comment]
Right, I have checked the latest Git version (081c017), and I can see the issues I pointed out have already been resolved, thanks.

**Comment by andrewshadura** [comment]
There’s some new stuff in #43 :)
